olive/passes/onnx/optimum_conversion.py

```diff
@@ -63,7 +64,7 @@ def validate_search_point(
         return True
 
     def _run_for_config(
-        self, model: OptimumModelHandler, data_root: str, config: Dict[str, Any], output_model_path: str
```

Comment:
I'm kinda confused now, since we removed optimum model, what is the difference between the onnxconversion pass and optimum conversion pass?

Reply:
Certain models need patching to export and optimum does that. Basically, the usual `torch.onnx.export` would not succeed for these models. OnnxConversion uses the `torch.onnx.export` and OptimumConversion uses the same with some patchwork to make a model _exportable_.

Reply:
Maybe out of the scope of this PR, but can we add patching feature to our onnx conversion pass?

Reply:
I don't think it's straightforward since there are lots of model specific information and configs required for such features. Optimum actively maintains it. 

With optimum conversion, we just have to provide the model name, task/feature. They do the conversion args and everything automatically. 

even our automatic conversion args is not up-to-date since new models get added everytime and huggingface doesn't maintain the `transformers.onnx` module we rely on. I have been considering migrating that feature to use optimum configs underneath when I have the time to try it.